# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1422044485 18000
#      Fri Jan 23 15:21:25 2015 -0500
# Branch stable
# Node ID 75a671fcdd734eca88b4ca90a940bed45263924c
# Parent  520ee3fec8eb2a70de1e6e0d7eac401980a4dbbb
* Faddeeva.cc: Also use gnulib::log.

diff --git a/liboctave/cruft/Faddeeva/Faddeeva.cc b/liboctave/cruft/Faddeeva/Faddeeva.cc
--- a/liboctave/cruft/Faddeeva/Faddeeva.cc
+++ b/liboctave/cruft/Faddeeva/Faddeeva.cc
@@ -206,16 +206,17 @@ static inline double my_copysign(double 
 // If we are using the gnulib <cmath> (e.g. in the GNU Octave sources),
 // gnulib generates a link warning if we use ::floor instead of gnulib::floor.
 // This warning is completely innocuous because the only difference between
 // gnulib::floor and the system ::floor (and only on ancient OSF systems)
 // has to do with floor(-0), which doesn't occur in the usage below, but
 // the Octave developers prefer that we silence the warning.
 #  ifdef GNULIB_NAMESPACE
 #    define floor GNULIB_NAMESPACE::floor
+#    define log GNULIB_NAMESPACE::log
 #  endif
 
 #else // !__cplusplus, i.e. pure C (requires C99 features)
 
 #  include "Faddeeva.h"
 
 #  define _GNU_SOURCE // enable GNU libc NAN extension if possible
 
